components/eamxx/src/physics/cosp/cosp_functions.cpp

```diff
@@ -0,0 +1,12 @@
+#include "cosp_functions.hpp"
+namespace scream {
+    namespace CospFunc {
+        void initialize(int ncol, int nsubcol, int nlay) {
+            std::cout << "brhdebug: call cosp_c2f_init()" << std::endl;
```

Comment:
On massively large simulations, this is going to inject lots of lines in e3sm.log. Perhaps you could pass a flag to the init function, telling it whether to print to screen or not?

Reply:
This needs to be removed sorry.

Reply:
Removed.